lib/archethic_web/graphql_schema/resolver.ex

```
@@ -112,7 +112,7 @@ defmodule ArchethicWeb.GraphQLSchema.Resolver do
     )
   end
 
-  def network_transactions(type, page) do
+  def network_transactions(type, page) when page >= 1 do
```

I think you should also return an error when the page is < 1

---

Introduced new `page_type`